test/src/main/tools/setupDB/README.md

```diff
@@ -2,10 +2,10 @@
 
 ## Getting Started
 
-You can see three folders
-    - IAF-MS-SQL
-    - IAF-Oracle
-    - IAF-MySQL
+There are three folders containing Docker configuration:
```

Comment:
configurations

Reply:
Ook misschien mooi te vermelding dat de standaard user testiaf_user is, en standaard wachtwoord testiaf_user00